examples/seismic/elastic/elastic_example.py

```diff
@@ -45,6 +45,13 @@ def run(shape=(50, 50), spacing=(20.0, 20.0), tn=1000.0,
     return rec1, rec2, vx, vz, txx, tzz, txz, summary
 
 
+def test_elastic():
```

Comment:
for the next PR...why don't we just put this in ``tests``?
same for the diffusion example...

Reply:
I think it's fine here, it's just running the example with an extra assert